src/lando/dockerflow/views.py

```diff
@@ -0,0 +1,84 @@
+from django.db import connection
+from django.db.utils import OperationalError
+from django.http import JsonResponse
+from django.views import View
+
+from lando.dockerflow.decorators import disable_caching, log_request
+
+
+class DockerflowView(View):
+    """
+    This is the base class view for all Dockerflow related views.
+
+    It handles common functionality needed by all Dockerflow views.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    @staticmethod
+    def _json_response(data, status):
+        return JsonResponse(data, status=status, json_dumps_params={"indent": 2})
+
+
+class VersionView(DockerflowView):
+    """
+    This view handles the version information of Lando.
+
+    It returns a JSON response containing the version information.
+    """
+
+    def get(self, request):
+        status = 200
```

Comment:
nit: I do think it's cleaner to not rewrite this variable if there is a way around it (i.e., treat it the same way as you are treating `data` below.)

Reply:
I was trying to follow the same pattern as `healthy = True` on #57, since I think you asked to move it outside the try/except there, too.

I'm fine with either, just trying to be consistent.

Reply:
Oh - yes definitely want it to be outside the block, but the nit was about setting it to one value then overriding that value (vs. setting it only once). Hopefully this makes sense. This also applies to line 57 btw!

Reply:
Done.